### PR TITLE
feat(docs): create mint module readme

### PR DESCRIPTION
**Pull request by barriebyron**
Hi team!
We were missing a basic readme for our custom mint module

### CODE REVIEW

File: x/mint/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Mint module
+
+The mint module takes on the task of handling the creation and destruction of tokens based on the current status of the circulating supply.
+The minting mechanism allows for a flexible inflation rate determined by market demand targeting a particular bonded-stake ratio and effects a balance between market liquidity and staked supply.
```
Comment (gsora) [inline x/mint/README.md]: Considering we're using a fixed minting schedule, this statement does not apply IMO

**Review by gsora — COMMENTED** [review]
Thank you for adding a README!

Added a comment.

File: x/mint/README.md (hunk 1 above)
Comment (barriebyron) [reply x/mint/README.md]: oh!? tell me more @gsora teach me!
I grabbed this description from the [DID conference paper](https://github.com/elesto-dao/conference-papers/tree/master/02_Elesto_ID) that I am reviewing for  @singhhp1069  

Let's fix both descriptions.

Comment (barriebyron) [reply x/mint/README.md]: hi friends! What changes are required here ? advise

Comment (gsora) [reply x/mint/README.md]: The mint module has a fixed minting supply which does not depend on any external factor. When a block gets created, a pre-defined amount of token gets minted.

This in turn means that the inflation rate is also fixed.

@noandrea do you agree?

**Comment by barriebyron** [comment]
@singhhp1069  can you update this description of the mint module that Elesto is using?

File: x/mint/README.md (hunk 1 above)
Comment (PaddyMc) [inline x/mint/README.md]:
```suggestion

```

Comment (PaddyMc) [reply x/mint/README.md]: we can just remove this part and we will be gtg!

**Comment by barriebyron** [comment]
thanks @PaddyMc ! welcome back

**Pull request merged**